public/index.html

```diff
@@ -4,6 +4,25 @@
     <meta http-equiv="x-ua-compatible" content="ie=edge" />
     <title>Screenshot as a service</title>
     <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <link
+      rel="apple-icon"
+      sizes="180x180"
+      href="%PUBLIC_URL%/apple-icon-180x180.png"
+    />
+    <link
+      rel="icon"
+      type="image/png"
+      href="%PUBLIC_URL%/favicon-32x32.png"
+      sizes="32x32"
+    />
+    <link
+      rel="icon"
+      type="image/png"
+      href="%PUBLIC_URL%/favicon-16x16.png"
+      sizes="16x16"
+    />
+    <meta name="description" content="A simple note taking app" />
+    <meta name="theme-color" content="#ffffff" />
```

Comment:
Please use this https://nextjs.org/docs/api-reference/next/head to add the `<link>`, and `<meta>` tag. 

The `index.html` wouldn't be used in next.js.

Otherwise, we can remove these lines.